loader/plugins/prune.js

```diff
@@ -0,0 +1,42 @@
+const CordlrPlugin = require('cordlr-plugin')
+
+module.exports = class HelpPlugin extends CordlrPlugin {
+  constructor (bot, config) {
+    super(bot, config)
+
+    this.name = 'Cordlr Core - Prune'
+    this.description = 'Cordlr Core plugin for pruning an specific amount of messages'
+
+    this.commands = {
+      'prune': {
+        'usage': '<x>',
+        'function': 'pruneMessages',
+        'description': 'Prunes the last x messages',
+        'permissions': [
+          'MANAGE_MESSAGES'

```

Comment:
Hopefully it is just me who has messed up my server-boilerplate development environment. I will try and delete and do a fresh install and test again later when I got time and then report back to you.